web3.md

```diff
@@ -587,5 +589,32 @@ WEB3 JSON RPC
     rule <k> #acctFromPrivateKey KEY => #newAccount #addrFromPrivateKey(KEY) ... </k>
          <accountKeys> M => M[#addrFromPrivateKey(KEY) <- #parseHexWord(KEY)] </accountKeys>
 
+    syntax KItem ::= "#firefly_addAccount"
+ // --------------------------------------
+    rule <k> #firefly_addAccount ... </k>
+         <params> [ (ADDRESS => #parseHexWord(ADDRESS)),
+                    (BALANCE => #parseHexWord(BALANCE)),
+                    (NONCE => #parseHexWord(NONCE)),
+                    (CODE => #parseByteStack(CODE)),
+                    ({ STORAGE } => #parseMap({ STORAGE })),
+                    .JSONList ]
+         </params>
+
+    rule <k> #firefly_addAccount => (#newAccount ADDRESS) ~> #loadAccountData ~> #sendResponse( "result": true ) ... </k>
```

Comment:
No need for parans around `#newAccound ADDRESS` I don't think.

Reply:
Also, we should handle the case that the account already exists, sending back an error.